app/models/post.rb

```diff
@@ -3,6 +3,10 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :title, presence: true, length: { maximum: 500 }
```

Comment:
- [ Required ] I think it would be better if you ensure that the `title` field does not exceed the `250` characters specified in the project requirements. Currently, you've set `title` field to have a maximum length of `500` characters which contravenes the project requirements. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-08-18 02-29-31](https://github.com/Enning94/Blog_App/assets/16392046/e5963f18-a925-45d2-9cef-a7bc5f8bb082)